apps/backoffice-v2/src/pages/Entity/hooks/useEntity/useEntity.tsx

```diff
@@ -112,15 +171,28 @@ export const useEntity = () => {
                     value: {
                       id,
                       title: `${category} - ${docType}`,
-                      data: Object.entries(propertiesSchema?.properties ?? {})?.map(
-                        ([title, { type, format, pattern, isEditable = true }]) => ({
+                      data: Object.entries(
+                        {
+                          ...additionProperties,
+                          ...propertiesSchema?.properties,
+                        } ?? {},
+                      )?.map(
+                        ([
                           title,
-                          value: properties?.[title] ?? '',
-                          type,
-                          format,
-                          pattern,
-                          isEditable: caseState.writeEnabled && isEditable,
-                        }),
+                          { type, format, pattern, isEditable = true, dropdownOptions, value },
+                        ]) => {
+                          const fieldValue = value || (properties?.[title] ?? '');
```

Comment:
Should either swap the positions of the condition or make a more explicit check to decide which value to use for example using `type`, `format`, or `dropdownOptions`.